aquality-automation/aquality-tracking-ui
# Features:

# PR Details

Rename Import Token to API Token 

## Related Issue

#23 

## How Has This Been Tested

1. Go to administration
1. See the API Token page instead of Import Token
1. Open API Token page
1. Try to generate token
1. Token is generated
1. Validate new page description

## Checklist
